constants/trainer_constants.asm

```diff
@@ -414,8 +408,8 @@ NUM_NONTRAINER_PHONECONTACTS EQU const_value - 1
 	const HUEY3
 
 	trainerclass SUPER_NERD ; 29
-	const STAN
-	const ERIC
+	const ERIC1 ; unused
+	const ERIC2
```

Comment:
`ERIC2` is used in GoldenrodUnderground; please change these to `ERIC_UNUSED` and `ERIC`, like Crystal's `NAOKO_UNUSED` and `NAOKO`.